packages/frontend/editor-ui/src/components/CanvasChat/future/components/DemoFooter.vue

```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+import LogsPanel from '@/components/CanvasChat/future/LogsPanel.vue';
+import { useSettingsStore } from '@/stores/settings.store';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { computed } from 'vue';
+
+const { isNewLogsEnabled } = useSettingsStore();
+const workflowsStore = useWorkflowsStore();
+const hasExecutionData = computed(() => workflowsStore.workflowExecutionData);
+</script>
+
+<template>
+	<LogsPanel v-if="isNewLogsEnabled && hasExecutionData" :is-read-only="true" />
```

Comment:
This footer is intended for the execution history page only. Is there any other conditions that I should check to _not_ show the logs panel?